src/components/Forum/ForumApprovedPostDetails/ForumPostDetailsList.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import {
+  Table, TableContainer, TableBody, TableRow, Typography, withStyles,
+  IconButton,
+} from '@material-ui/core';
+import { withTheme } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import MuiTableCell from '@material-ui/core/TableCell';
+import DeleteIcon from '@material-ui/icons/Delete';
+import EditIcon from '@material-ui/icons/Edit';
+import ReplyIcon from '@material-ui/icons/Reply';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const labelsColumnWidth = 160;
```

Comment:
Does the width have to be a constant value? Can it be relative?

Reply:
Relative is probably better. Will change.